packages/core/admin/admin/src/content-manager/components/AttributeFilter/hooks/useAllowedAttributes.js

```diff
@@ -32,9 +47,13 @@ const useAllowedAttributes = (contentType, slug) => {
         return false;
       }
 
+      if (CREATOR_ATTRIBUTES.includes(attr) && !canReadAdminUsers) {
+        return false;
+      }
```

Comment:
@Marc-Roig I'd be interested in your opinion here, because I lack some knowledge: I think at the moment the creator fields are excluded from RBAC, so that e.g. users who can't see all fields of a content-type, can always see who has created/ edited an entity.

From what I understand now is that we are changing this, so that these fields are covered by RBAC. That also means we can not assume a user has always access to them?

If so then this seems to be a big deal to me and I think I'd like to involve others to discuss the implications of that ...

Reply:
Adding @innerdvations into the discussion. I do think creator attributes should be seen by everybody by default too. If I recall correctly the problem appears when you make those fields filterable.

Nevertheless, apart from people not seeing the fields, which other problems do you think could appear here? I can only think of Audit logs not showing the creator fields now.

Reply:
> Nevertheless, apart from people not seeing the fields, which other problems do you think could appear here? I can only think of Audit logs not showing the creator fields now.

I don't know tbh, but from discussions with Alex I had the feeling these fields were deliberately excluded from RBAC, because even if a user doesn't have access to other users, it might still be important to understand who has created/ edited something else.

I think I would prefer:

- @Feranchz if you could document in the PR description the discussion you had about that to understand the thinking behind it
- if this change could be documented using an ADR, which can then be discussed in the sync meetings

This feels like a small change, but I have the feeling there are widespread implications and I'd like to make sure the whole team is on board with that.

Reply:
I agree we need an ADR or RFC for this, which should very clearly define:

- if `visible` just means hidden from display, those fields must still be shown in RBAC or there needs to be an explicit private/public [the direct problem this PR is solving]
- what `private` and `visible` mean
- should we add a `readable` or `public` field that means it's always available regardless of permissions?
- which creator fields should be publicly available (or not)?
- based on all of that, what should appear in the RBAC permissions?

Reply:
creator fields should fall under rbac.
Auditlogs could just bypass rbac to get the right information (Since it is a system tool and not user)

My reasoning is it adds an extra security layer aka you can then just say with rbac the following fields this user should never be allowed to populate example would be resetToken then instad of having only the the 1 layer of it being senatized you have 2 secutiy layers. what means if one fails the other will still be there. like if rbac is setup properly aka it checks populates filters and sorts then RBAC is a good second layer to have over everything.

Reply:
We might remove the extra security layer if we do bypasses imo, and make it confusing for us. I agree that having the creator fields in RBAC + all other non visible fields could be a great idea, and we need to see the implications of it.

Reply:
@Marc-Roig Not how I see it
auditlogs is done by the system and not the user.
RBAC is for user permission
so system tasks bypass RBAC

Reply:
RBAC is one part that could be affected for this, but we might have more. And adding bypasses each time we need to show creator fields might lead to unexpected data leaks.

Reply:
Agree they should fall under RBAC, there are many cases in which creator fields should be completely private and not available in the api response. We need to determine if we need to change anything about how it's currently working that could be a breaking change and have it all written down clearly (because right now it isn't)

Reply:
Again you don't do that.
You make a diffrence between user and system actions.

all system actions bypass RBAC fully since you have full cotrol over everying example would be auditlogs.
You would still use RBAC for the edit views since that is a user action and not system.